homeassistant/components/vegehub/sensor.py

```diff
@@ -0,0 +1,87 @@
+"""Sensor configuration for VegeHub integration."""
+
+from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_MAC, UnitOfElectricPotential
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MANUFACTURER, MODEL
+from .coordinator import VegeHubCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up Vegetronix sensors from a config entry."""
+    sensors: list[VegeHubSensor] = []
+    mac_address = config_entry.data[CONF_MAC]
+    num_sensors = config_entry.runtime_data.hub.num_sensors
+    num_actuators = config_entry.runtime_data.hub.num_actuators
```

Comment:
```suggestion
    hub = config_entry.runtime_data.hub
```
and access from the hub below (might be preference, but I think it's more readable)